AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -92,8 +92,8 @@ public class CardComponent: CardPublicKeyConsumer,
     ///   - cardPublicKeyProvider: The card public key provider
     ///   - style: The Component's UI style.
     internal init(paymentMethod: AnyCardPaymentMethod,
```

Comment:
the documentation for this method is missing some parameters, but this is `internal`, so maybe not important